app/workers/audio_callback_worker.rb

```diff
@@ -41,6 +42,9 @@ def perform(_sqs_msg, job)
 
     Shoryuken.logger.info("Updating #{job['type']}[#{audio.id}]: status => #{audio.status}")
     audio.save!
+    # announce the audio changes using the story as resource.
+    announce(:audio, :update, id: audio.story.id, resource: audio.story)
```

Comment:
So this would fire off an resource=story announcement on the `audio` channel?

Yeah, I _think_ it should be `announce(:story, :update, id: story.id, audio.story)`.